synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -148,7 +166,7 @@ export class SynopsysToolsParameter {
     // Wrap the file path with double quotes, to make it work with directory path with space as well
     stateFilePath = '"'.concat(stateFilePath).concat('"');
 
-    taskLib.debug("Generated state json file content is - ".concat(inputJson));
+    taskLib.debug("Generated state json file at - ".concat(stateFilePath));
```

Comment:
I think this debug will not help user as the file get's deleted after execution is completed.

Reply:
Similar things were already there for Coverity and BlackDuck, that's why I have added this in Polaris section